[BUGFIX] Pause Menu and Stickers have same zoom as HUD

gamerbross opened the pull request:
(https://github.com/FunkinCrew/funkin.assets/pull/7 required!!!)
The Pause Menu and Stickers Transition were using camHUD, which its zoom can change, scaling these without having to.
I added `camOther` as another camera which contains video cutscenes, dialogues, pause menu and sticker transition (or whatever that shouldn't have the same zoom as HUD)
I also removed `camCutscene`, because can be replaced with this new one.

Here I'm going to attach some issues caused by unintentional zoom.
(M.I.L.F. is a great song to use, as it zooms the HUD more)
---
![image](https://github.com/FunkinCrew/Funkin/assets/55158797/2ac109ad-f62d-48be-9f6d-b927e83bb771)
_Pause Menu getting cut by the HUD zoom._

https://github.com/FunkinCrew/Funkin/assets/55158797/dc49a9e9-ed41-43e2-bf7a-a2d2b841d855

_Sticker Transition zooming out a little bit_

Matriculaso commented:
Delete this shit bro

Matriculaso commented:
Pretty Useful tbh, I had an issue with stickers cuz I used a lot of camZoomings events and stickers were pretty messed up along with pause menu, Thanks Gamerbross!

gamerbross commented:
Ok it has some issues that I didn't saw because when I was trying it, camCutscene still existed, so there were no problems

gamerbross commented:
Just fixed those issues (now needs this pr in funkin.assets https://github.com/FunkinCrew/funkin.assets/pull/7)
(it also fixed one bug in the roses end cutscene!!!)
![image](https://github.com/FunkinCrew/Funkin/assets/55158797/ca5b9f13-1ad3-42d5-8d59-3f42bc0163ec)
Pause menu was on camHUD, so this red camera flash on camCutscene covered it, putting everything red.

Matriculaso commented:
hurry up and add this shit my grilfriend feet sticker pack needs this!!! feet is too big because of the zoom I do on her feet helpp pleass 😫😫😫😫

Stilic commented:
the renaming isn't needed.
it could cause breaking with existing scripts.

DMMaster636 commented on source/funkin/play/PlayState.hx:
```diff
@@ -491,9 +491,9 @@ class PlayState extends MusicBeatSubState
+   * The camera which contains, and controls visibility of, a video cutscene, dialogue, pause menu and sticker transition.
    */
-  public var camCutscene:FlxCamera;
+  public var camOther:FlxCamera;
```
Why replace camCutscene?? That can lead to some broken scripts.
Wouldn't it be smarter to just add a new camera for this?

EliteMasterEric replied:
That's true, it also breaks a bunch of cutscene scripts and those would need to be redone.

gamerbross replied:
I renamed it because it didn't just handle cutscenes now, but I just reverted it

gamerbross commented:
I reverted the camCutscene rename, so now the pull request in assets isn't really required, just removes unnecessary visibility changes to the camera

Matriculaso commented:
![image](https://github.com/FunkinCrew/Funkin/assets/88761489/8ffd1799-6be9-406e-b0e8-eff13cd2854e)

EliteMasterEric reviewed (approved):
Works good for me
